releasenotes/notes/simplify-textfiletodocument-c9d2fb7ed2c848ed.yaml

```diff
@@ -0,0 +1,3 @@
+preview:
+  - Remove most parameters from TextFileToDocument to make it match all other converters.
+  - Add support for ByteStreams
```

Comment:
There are 3 release notes, let's keep just one

Reply:
Woah what happened?? I was not really in my best shape while working on this, but that's another level :joy: